src/ch/jamiete/hilda/Hilda.java

```diff
@@ -133,7 +109,7 @@ public String getUsername() {
         return this.bot.getSelfUser().getName();
     }
 
-    public void start() {
+    void start() {
```

Comment:
You should not have removed the scope

Reply:
Please see https://stackoverflow.com/questions/215497/in-java-difference-between-package-private-public-protected-and-private.

Reply:
If you are trying to change the scope of methods in the main file you should do so in a different commit.